pkg/client/unversioned/request.go

```diff
@@ -641,6 +660,12 @@ func (r *Request) Stream() (io.ReadCloser, error) {
 // fn at most once. It will return an error if a problem occurred prior to connecting to the
 // server - the provided function is responsible for handling server errors.
 func (r *Request) request(fn func(*http.Request, *http.Response)) error {
+	//Metrics for total request latency
```

Comment:
I'm really not sure-- what is the goal of this latency measurement?

On Wed, Nov 11, 2015 at 10:43 AM, jay vyas <notifications@github.com> wrote:

> In pkg/client/unversioned/request.go
> <https://github.com/kubernetes/kubernetes/pull/16271#discussion_r44568576>
> :
>
> > @@ -641,6 +660,12 @@ func (r *Request) Stream() (io.ReadCloser, error) {
> >  // fn at most once. It will return an error if a problem occurred prior to connecting to the
> >  // server - the provided function is responsible for handling server errors.
> >  func (r *Request) request(fn func(*http.Request, *http.Response)) error {
> > +	//Metrics for total request latency
>
> the current code is chunking retries already. Maybe separate issue? Or
> should I fix it all in here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16271/files#r44568576>.
>